homeassistant/components/alarm_control_panel/mqtt.py

```diff
@@ -166,3 +166,7 @@ def alarm_arm_away(self, code=None):
                          self._payload_arm_away, self._qos)
         else:
             _LOGGER.warning("Wrong code entered while arming away!")
+
+    def alarm_trigger(self, code=None):
```

Comment:
Since it is not implemented for MQTT, do not override it and have the default behavior raise an exception.